src/commands/send.rs

```diff
@@ -169,6 +175,126 @@ fn print_response(blob: Vec<u8>, method_name: String) -> AnyhowResult {
             let response = Decode!(blob.as_slice(), ManageNeuronResponse)?;
             println!("Response: {:?\n}", response);
         }
+        SupportedResponse::SnsCanistersSummary => {
```

Comment:
Were you able to get an example of the output of each of these commands? It's a little hard to judge what it will look like based on these methods. If you haven't run a local sns yet, lets meet tomorrow and go through the runbook together